src/index.html

```diff
@@ -11,6 +11,22 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#" class="search-form">
+      <input
+        type="text"
+        name="searchQuery"
+        class="search-form__input search-form--big-input"
+        placeholder="Try &#34;Los Angeles&#34;"
+        data-qa="big"
+      >
+
+      <input
+        type="text"
+        name="searchQuery"
+        class="search-form__input search-form--small-input"
+        placeholder="Try &#34;Los Angeles&#34;"
+        data-qa="small"
+      >
+    </form>
```

Comment:
Each search bar be inside its own form element.
https://github.com/mate-academy/layout_search-bar-airbnb#tips--hints